### PR TITLE
Bug 1332448 - Remove taskrequest counter and timeout from seta

### PR DESCRIPTION
**Pull request by rwood-moz**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/2103)

### CODE REVIEW

File: treeherder/seta/models.py [hunk 1]
```diff
@@ -61,7 +36,6 @@ def adjust_jobs_priority(self, high_value_jobs, priority=1, timeout=0):
             elif jp.priority != priority:
                 logger.info('Increasing priority of {}'.format(jp.unique_identifier()))
                 jp.priority = priority
-                jp.timeout = timeout
                 jp.save(update_fields=['priority', 'timeout'])
```
Comment (jmaher) [inline treeherder/seta/models.py]: here you have 'timeout' in a field to be updated, should that be removed?

File: treeherder/seta/migrations/0003_remove_jobpriority_timeout.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-01-24 19:01
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```
Comment (edmorley) [inline treeherder/seta/migrations/0003_remove_jobpriority_timeout.py]: Could you combine the multiple migrations in this PR into one? :-)

File: treeherder/seta/models.py (hunk 1 above)
Comment (rwood-moz) [inline treeherder/seta/models.py]: Yes, missed that one thanks!

File: treeherder/seta/migrations/0003_remove_jobpriority_timeout.py (hunk 2 above)
Comment (rwood-moz) [inline treeherder/seta/migrations/0003_remove_jobpriority_timeout.py]: Ok, good call, thanks!

**Pull request merged**